cloudant-http/src/main/java/com/cloudant/http/internal/interceptors/IamCookieInterceptor.java

```diff
@@ -32,9 +32,10 @@
     public static final String IAM_TOKEN_SERVER_URL_PROPERTY_KEY = "com.cloudant.client.iamserver";
     public final URL iamServerUrl;
     private final byte[] tokenRequestPayload;
+    private final URL proxyURL;
```

Comment:
Not needed, as it is in the `CookieInterceptorBase`

Reply:
Fixed in a1c95fad4ccf0e845fe4649613fc184c33168220